src/synchronizeContacts.js

```diff
@@ -217,13 +217,7 @@ const synchronizeContacts = async (
           await cozyUtils.client.save(mergedContact)
           result.cozy.created++
         } else if (action === SHOULD_UPDATE) {
-          const cozyRemoteRev = get(cozyContact, [
-            'cozyMetadata',
-            'sync',
-            contactAccountId,
-            'remoteRev'
-          ])
-          if (googleContact.etag !== cozyRemoteRev) {
+          if (googleContact) {
```

Comment:
Do we need the check at all? We know there is a `googleContact `, and it's not used inside the condition anyway.

Reply:
true ! Just removed it and the tests are still green :-)